drivers/timer/nrf_grtc_timer.c

```diff
@@ -399,16 +358,22 @@ int z_nrf_grtc_wakeup_prepare(uint64_t wake_time_us)
 	nrfx_err_t err_code;
 	static uint8_t systemoff_channel;
 	uint64_t now = counter();
+	nrfx_grtc_sleep_config_t sleep_cfg;
 	/* Minimum time that ensures valid execution of system-off procedure. */
-	uint32_t minimum_latency_us = nrfy_grtc_waketime_get(NRF_GRTC) +
-				      nrfy_grtc_timeout_get(NRF_GRTC) +
-				      CONFIG_NRF_GRTC_SLEEP_MINIMUM_LATENCY;
+	uint32_t minimum_latency_us;
 	uint32_t chan;
 	int ret;
 
+	nrfx_grtc_sleep_configuration_get(&sleep_cfg);
+	minimum_latency_us = (sleep_cfg.waketime + sleep_cfg.timeout) * USEC_PER_SEC / 32768 +
```

Comment:
Is this magic number avoidable? Could it be taken from a define somewhere?

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/71688#discussion_r1576108602
IMO define suggests that the parameter is somehow configurable, while it is not

Reply:
I agree with https://github.com/zephyrproject-rtos/zephyr/pull/71688/files#r1582781660
This should be a define at the beginning of file at least, ideally taken from HW configuration in the future.